src/routes/+page.svelte

```diff
@@ -269,11 +266,11 @@
     align-items: center;
   }
 
-  .inner-content img {
+  /* .side-by-side-img {
```

Comment:
Deleteeeee it!!! i've been removing classes like this when i replace with tailwind which is very satisfying